en/rulesets/standard-ruleset.rst

```diff
@@ -1,78 +1,39 @@
-
 Every ``iati-activity`` or ``iati-organisation`` record has the option to use different elements of the IATI Standard, and to also repeat certain elements, so in defining a Standard Ruleset it is necessary to include rules that may or may not need to apply for each individual ``iati-activity`` or ``iati-organisation`` record.
-
 If we take a ``transaction`` from an ``iati-activity`` record as an example, we can see that this element does not have to be reported. However, it may also be reported many times. There are rules in the Standard Ruleset to check that some of the data reported in transactions makes sense, so these tests could either be run many times, or not at all on a single ``iati-activity`` record. 
 
 Hence, the minimum and maximum number of tests undertaken on an ``iati-activity`` or ``iati-organisation`` can fluctuate according to the content.
 
 Non-Machine Readable Rules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-reporting-org/\@ref
+IATI Org IDs\@ref
 -------------------
-
-* The agency prefix MUST be a valid code in the IATI code list
-* The identifier MUST be the same as that recorded by the publisher on the IATI Registry
-
-participating-org/\@ref
------------------------
-
-* When using this attribute, it must be a valid organisation identifier.
-
-transaction/provider-org/\@ref
-------------------------------
-
-* When using this attribute, it must be a valid organisation identifier.
-
-transaction/receiver-org/\@ref
-------------------------------
-
-* When using this attribute, it must be a valid organisation identifier.
-
+* When reporting an IATI Org ID either for the publishers or another agency, it must be a valid organisation identifier.
+* For the publisher, the identifier MUST be the same as that recorded by the publisher on the IATI Registry
 iati-identifier
 ---------------
-
 * It MUST be globally unique among all activities published through the IATI Registry
 * Once an activity has been reported to IATI its identifier MUST NOT be changed in subsequent updates
 * It MUST be prefixed with EITHER the organisation identifier found in ``reporting-org/@ref`` OR a previous ``reporting-org identifier`` reported in ``other-identifier``
 
-budget
-------
-
-* The budget element should be repeated for each year of the activity's duration
-* Each budget should cover a period no longer than 1 year
-
-sector
-------
-
-* Sector MUST be reported at either activity level or transaction level but not both
-* At transaction level, only one sector per vocabulary should be reported
-
 recipient-country and recipient-region
 --------------------------------------
-
-* Either recipient-country or recipient-region MUST be reported at activity level or transaction level but not both.
-
+* The recipient region must only be published in addition to the recipient country.
```

Comment:
@amy-silcock Where does this rule come from?

I’m not aware of this rule. It’s not my understanding of the way `recipient-region` works.

Reply:
+1 this is contra my understanding of recipient regions & countries, so it would be good to have some consultation on it. I'd argue that it would create a lot of redundancy in hierarchical activity models (wherein there's a regional high level activity with multiple national activities underneath (some recent HUM publishers do this).

Reply:
There are a few different rules and guidelines around reporting recipient countries and regions. The part we can't check is: 'Recipient-region **must** not be used merely to describe the region of a country reported in recipient-country, but ONLY if the region is a recipient IN ADDITION to the country.'

In my understanding this is to say if your activity benefits country x 100%, you must not also add the region it is within.

Perhaps it's my wording that's unclear?

Reply:
Thanks Amy – that clarifies things. That’s my understanding of the rule in question, too, so I agree that there’s a problem with the wording.

> recipient region must only be published **in addition to** the recipient country

^^ This wording seems to suggest `recipient-region` can’t appear on its own, but only when accompanied by a `recipient-country`.

However, I think the main problem here is that this rule shouldn’t be included with the standard ruleset, because as you correctly say, it’s not possible to check using computer code. The standard ruleset is a collection of rules "[that a computer could, in principle, test.](http://reference.iatistandard.org/203/rulesets/standard-ruleset/#standard-ruleset)" This one can’t be, so it shouldn’t be included here.

Reply:
Thanks @andylolz I'm proposing to include it in the 'non-machine readable rules' section for exactly that reason. These are rules which a computer can't (at least can't without additional information) check.

Reply:
The name is confusing, but the "non-machine readable" rules **can** be checked by a computer.

The [standard ruleset page](http://reference.iatistandard.org/203/rulesets/standard-ruleset/) says:

> The Standard Ruleset is a collection of rules that cover some of the basic requirements of the IATI Standard and **that a computer could, in principle, test.**

Here’s what it says about non-machine readable rules:

> Some rules do not fit into this format, we call them “Non-Machine Readable Rules”. **It should be possible to write bespoke computer code to test each rule, but it requires a bit more work, both in terms of understanding what the rule means and the effort required to check.**

Also worth taking a look at #347. Hayden states that some rules “[…] cannot be 'easily' machine-checked, so would not fit within the [Standard Ruleset](http://iatistandard.org/202/rulesets/standard-ruleset/) as it is currently defined”. I think that’s also the case for this rule.